libr/bin/format/pe/pe.c

```diff
@@ -647,6 +647,65 @@ int PE_(bin_pe_get_actual_checksum)(struct PE_(r_bin_pe_obj_t)* bin) {
 	return computed_cs;
 }
 
+static void update_if_sum_larger_and_within_file(ut32 offset, ut32 size, ut32 file_size, ut32* largest_offset, ut32* largest_size) {
+	if (offset+size <= file_size && offset+size > (*largest_offset+*largest_size)) {
+		*largest_offset = offset;
+		*largest_size = size;
+	}
+}
+
+/* Inspired from https://github.com/erocarrera/pefile/blob/master/pefile.py#L5425 */
+int PE_(bin_pe_get_overlay)(struct PE_(r_bin_pe_obj_t)* bin, ut32* size) {
```

Comment:
what about ut64 instead of ut32 here? i have the feeling that this will not work on PE64